src/PublicAssets.php

```diff
@@ -12,20 +12,6 @@ final class PublicAssets {
 	public static function enqueue_js() {
 		$js_creation = filectime( get_template_directory() . '/assets/build/index.js' );
 
-		$jquery_should_wait = is_plugin_active( 'planet4-plugin-gutenberg-blocks/planet4-gutenberg-blocks.php' ) && ! is_user_logged_in();
-
-		$jquery_deps = $jquery_should_wait ? [ 'planet4-blocks-script' ] : [];
-
-		// JS files.
-		wp_deregister_script( 'jquery' );
```

Comment:
The version in WP will still be included if you don't remove the dependency on l.41/27 when registering index.js